xml/System.Text.Json/JsonProperty.xml

```diff
@@ -211,9 +211,13 @@ This method is functionally equal to doing an ordinal comparison of `text` and <
         <Parameter Name="writer" Type="System.Text.Json.Utf8JsonWriter" />
       </Parameters>
       <Docs>
-        <param name="writer">To be added.</param>
-        <summary>To be added.</summary>
+        <param name="writer">The writer to which to write the property.</param>
+        <summary>Writes this property into the specified writer as a named JSON object property.</summary>
```

Comment:
Keeping this consistent with the recent @rpetrusha modifications to similar PRs:

```suggestion
        <summary>Writes the property to the provided writer as a named JSON object property.</summary>
```

Reply:
Sounds good!